File: benchmark/PHP 7.0/循环分支累加耗时.php
<?php

$t = microtime(true);

$sum = 0;
for ($i = 0; $i < 10000000; ++$i)
{
	if ($i <= 2500000)
		$sum += $i;
	else if ($i <= 5000000)
		$sum += $i+1;
	else if ($i <= 7500000)
		$sum += $i+2;
	else
		$sum += $i+3;
}
echo $sum, "\n";

print (microtime(true) - $t);

File: benchmark/CZL 1.0/循环分支累加耗时.c
main
{
	int i, sum=0
	
	for i in (0, 10000000-1)
		if i <= 2500000
			sum += i
		else if i <= 5000000
			sum += i+1
		else if i <= 7500000
			sum += i+2
		else
			sum += i+3
		
	echo(sum)
	echo(clock())
}
